src/integTest/groovy/net/wasdev/wlp/gradle/plugins/AbstractIntegrationTest.groovy

```diff
@@ -78,11 +78,10 @@ abstract class AbstractIntegrationTest {
         GradleConnector gradleConnector = GradleConnector.newConnector()
         gradleConnector.forProjectDirectory(projectDir)
         ProjectConnection connection = gradleConnector.connect()
-
         try {
             BuildLauncher build = connection.newBuild()
             build.setJvmArguments("-DWLP_DIR=$WLP_DIR")
-            build.withArguments("-i");
+            build.withArguments("-Prunit=online");
```

Comment:
Set this in gradle.properties and we can pass in the property to overwrite it if we want.